cpp/tests/spatial/distance/pairwise_multipoint_equals_count_test.cu

```diff
@@ -0,0 +1,157 @@
+/*
```

Comment:
This file seems to be a duplicate of the one in the `equality` directory. Perhaps copied here accidentally?